packages/acs-ui-common/review/beta/acs-ui-common.api.md

```diff
@@ -85,6 +85,9 @@ export type TelemetryEvent = {
     data?: Record<string, unknown>;
 };
 
+// @internal
+export type _TelemetryImplementationHint = 'Call' | 'Chat' | 'CallWithChat' | 'StatefulComponents';
```

Comment:
can we review if we want to use these enums with the team?

I am on the fence about StatefulComponents but im not sure what we should be doing instead.

CallComposite, ChatComposite, CallWithChatComposite, and then default for components?